tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_test.ts

```diff
@@ -247,5 +247,172 @@ describe('tb_feature_flag_data_source', () => {
         );
       });
     });
+
+    describe('persistFeatureFlags', () => {
+      it('calls getPersistentFeatureFlags and stores new flag', () => {
+        const getPersistentFeatureFlagsSpy = spyOn(
```

Comment:
I think these three persistFeatureFlags tests would be better without the getPersistentFeatureFlagsSpy. They would still offer all the test coverage you'd want and they'd be more readable.

Here, for example, you just need to rename and delete some lines of code:
```
      it('sets flag values', () => {
        spyOn(localStorage, 'getItem').and.returnValue(null);
        const setItemSpy = spyOn(localStorage, 'setItem').and.stub();
        dataSource.persistFeatureFlags({enabledScalarDataTable: true});
        expect(setItemSpy).toHaveBeenCalledOnceWith(
          'tb_feature_flag_storage_key',
          '{"enabledScalarDataTable":true}'
        );
```

Reply:
Understood. I removed those checks. I also removed them from the resetPersistedFeatureFlag test.